src/App.js

```diff
@@ -0,0 +1,15 @@
+import './App.css';
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <div className="App">
+
+      <div className="Calculator">
+        <Calculator />
+      </div>
+    </div>
```

Comment:
- [ ] Good job working on Math magicians: components 🥇, however, make you stick to the given design as it is [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A) 

> | | Given design | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-08-23 142640](https://github.com/paulshaun-kdtk/React_Math_Magicians/assets/57408419/2d949127-98a7-424e-b303-8b87c87ff972)| |![Screenshot 2023-08-23 142534](https://github.com/paulshaun-kdtk/React_Math_Magicians/assets/57408419/9a797010-749a-42e1-98d3-9edb75792c6e)| | 


Kindly fix it 👍